get_files_from_drive.py

```diff
@@ -0,0 +1,31 @@
+import argparse
```

Comment:
I don't think we should provide this since we'd have to maintain it if Google changes things.

Reply:
So what should I do with it? It's kinda needed for the travis.yml for testing right? Should I just copy the code into the yml?

Reply:
You are correct. nvmd